src/js/constants.js

```diff
@@ -0,0 +1,17 @@
+
+export const kbEngLower = ['\`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace', 'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'Del', 'Caps Lock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'Enter', 'Shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', '↑', 'Shift', 'Ctrl', '❖', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→', 'Info'];
+
+export const kbEngUpper = ['~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', 'Backspace', 'Tab', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|', 'Del', 'Caps Lock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '\"', 'Enter', 'Shift', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?', '↑', 'Shift', 'Ctrl', '❖', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→', 'Info'];
+
+export const kbRusLower = ['ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace', 'Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'Del', 'Caps Lock', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', 'Enter', 'Shift', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '.', '↑', 'Shift', 'Ctrl', '❖', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→', 'Info'];
+
+export const kbRusUpper = ['Ё', '!', '\"', '№', ';', '%', ':', '?', '*', '(', ')', '_', '+', 'Backspace', 'Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '/', 'Del', 'Caps Lock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'Enter', 'Shift', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', ',', '↑', 'Shift', 'Ctrl', '❖', 'Alt', '', 'Alt', 'Ctrl', '←', '↓', '→', 'Info'];
+
+export const kbClass = [' Backquote', ' Digit1', ' Digit2', ' Digit3', ' Digit4', ' Digit5', ' Digit6', ' Digit7', ' Digit8', ' Digit9', ' Digit0', ' Minus', ' Equal', ' -functional -backspace Backspace', ' -functional -tab Tab', ' KeyQ', ' KeyW', ' KeyE', ' KeyR', ' KeyT', ' KeyY', ' KeyU', ' KeyI', ' KeyO', ' KeyP', ' BracketLeft', ' BracketRight', ' Backslash', ' -functional -del Delete', ' -functional -caps CapsLock', ' KeyA', ' KeyS', ' KeyD', ' KeyF', ' KeyG', ' KeyH', ' KeyJ', ' KeyK', ' KeyL', ' Semicolon', ' Quote', ' -functional -enter Enter', ' -functional -lshift ShiftLeft', ' KeyZ', ' KeyX', ' KeyC', ' KeyV', ' KeyB', ' KeyN', ' KeyM', ' Comma', ' Period', ' Slash', ' -arrow ArrowUp', ' -functional -rshift ShiftRight', ' -functional -lctrl ControlLeft', ' -functional -win MetaLeft', ' -functional AltLeft', ' -functional -space Space', ' -functional AltRight', ' -functional ControlRight', '  -arrow ArrowLeft', ' -arrow ArrowDown', ' -arrow ArrowRight', ' -functional -info'];
```

Comment:
you are right here, the best practices is to use constants. But lets go further - these objects/array are still no _consant_. To make them 100% constant you can use Object.freeze() method. Pls check how it works.

Reply:
Is this what you want?

![image](https://user-images.githubusercontent.com/44834982/95052422-9582d400-06f7-11eb-945f-079a6d3eafb3.png)

Reply:
yes, that is fine :)